### ISSUE TITLE
Doc fix

### ISSUE THREAD

**Issue by peoplemerge** [dwhjames/aws-wrap#25]
Would love to send a pull request for doc but I don't see it here and not sure how to request src for github page.

http://pellucidanalytics.github.io/aws-wrap/doc/dynamodb.html
add:
`import com.pellucid.wrap.dynamodb._`
replace
```
 override def fromAttributeMap(
      item: mutable.Map[String, AttributeValue]) =
```
with
```
      item: collection.mutable.Map[String, AttributeValue]) =
```

**Comment by dwhjames-pellucid** [comment]
Ah, you’ll need to make your PR against the `gh-pages` branch
https://github.com/pellucidanalytics/aws-wrap/tree/gh-pages

**Comment by dwhjames** [comment]
Closed by #26
